source/github-helpers/dom-formatters.tsx

```diff
@@ -20,10 +20,10 @@ export const codeElementsSelector = [
 ];
 
 export function shortenLink(link: HTMLAnchorElement): void {
-	// Exclude the link if the closest element found is not `.comment-body`
+	// Exclude the link if the closest element found is not `.markdown-body`
 	// This avoids shortening links in code and code suggestions, but still shortens them in review comments
 	// https://github.com/refined-github/refined-github/pull/4759#discussion_r702460890
-	if (link.closest(String([...codeElementsSelector, '.comment-body']))?.classList.contains('comment-body')) {
+	if (link.closest(String([...codeElementsSelector, '.markdown-body']))?.classList.contains('markdown-body')) {
```

Comment:
Make sure that this still works on the old version. See context in

https://github.com/refined-github/refined-github/pull/5472#issuecomment-1065227616

The link mentioned in the issue was added by https://chromewebstore.google.com/detail/octolinker/jlmafbaeoofdegohdhinkhilhclaklkp?hl=en&pli=1

Reply:
> Make sure that this still works on the old version. See context in https://github.com/refined-github/refined-github/pull/5472#issuecomment-1065227616

It works here.

<img width="503" alt="image" src="https://github.com/user-attachments/assets/71b1fc50-208c-42ad-9de8-232857caab01">

Reply:
>The link mentioned in the issue was added by [chromewebstore.google.com/detail/octolinker/jlmafbaeoofdegohdhinkhilhclaklkp?hl=en&pli=1](https://chromewebstore.google.com/detail/octolinker/jlmafbaeoofdegohdhinkhilhclaklkp?hl=en&pli=1)

Here also works.

<img width="1382" alt="image" src="https://github.com/user-attachments/assets/5aedc7db-37f9-4799-90b7-7272d22129cd">

Reply:
Those links are not added by octolinker, they're already excluded by `linkifiedURLClass`

Reply:
> It works here.

This also doesn't contain any links, it was just an example of the current DOM structure.

Reply:
It seems that [OctoLinker is broken](https://togithub.com/OctoLinker/OctoLinker/issues/1907). I'm just testing this PR manually.